rj_gameplay/rj_gameplay/tactic/clear_tactic.py

```diff
@@ -0,0 +1,73 @@
+from dataclasses import dataclass
+from typing import List, Optional
+from typing import Dict, Generic, List, Optional, Tuple, Type, TypeVar
+
+import stp.action as action
+import stp.rc as rc
+import stp.tactic as tactic
+import stp.role as role
+
+import rj_gameplay.eval
+import rj_gameplay.skill as skills
+from rj_gameplay.skill import pivot_kick, receive
+import stp.skill as skill
+import numpy as np
+
+class ClearerCost(role.CostFn):
+    """
+    A cost function for how to choose a robot that will clears
+    """
+    def __call__(self,
+                robot:rc.Robot,
+                prev_result:Optional["RoleResult"],
+                world_state:rc.WorldState) -> float:
+        if not robot.visible:
+            return 99999
+        else:
+            return np.linalg.norm(world_state.ball.pos - np.array(robot.pose[0:2]))
+
+class Clear(tactic.ITactic):
```

Comment:
```suggestion
# TODO: discuss if clear deserves its own tactic if we can just use pivot_kick skill for it
class Clear(tactic.ITactic):
```

Reply:
I think it does, because ideally there would be some sort of heuristic function that chooses a good place to pass to.

Reply:
ah okay